trulens_eval/trulens_eval/ux/styles.py

```diff
@@ -1,12 +1,63 @@
-from trulens_eval.feedback import default_pass_fail_color_threshold
+import numpy as np
 
-# These would be useful to include in our pages but don't yet see a way to do this in streamlit.
+from trulens_eval.util import SerialModel
+
+
+class CATEGORY:
+    """
+    Feedback result categories for displaying purposes: pass, warning, fail, or
+    unknown.
+    """
+
+    class Category(SerialModel):
+        name: str
+        adjective: str
+        threshold: float
+        color: str
+        icon: str
+
+    PASS = Category(
+        name="pass", adjective="high", threshold=0.8, color="#aaffaa", icon="✅"
+    )
+    WARNING = Category(
+        name="warning",
+        adjective="medium",
+        threshold=0.6,
+        color="#ffffaa",
+        icon="⚠️"
+    )
+    FAIL = Category(
+        name="fail", adjective="low", threshold=0.0, color="#ffaaaa", icon="🛑"
+    )
+    UNKNOWN = Category(
+        name="unknown",
+        adjective="unknown",
+        threshold=np.nan,
+        color="#aaaaaa",
+        icon="?"
+    )
+
+    ALL = [PASS, WARNING, FAIL] # not including UNKNOWN intentionally
+
+    @staticmethod
+    def of_score(score: float) -> Category:
+        for cat in CATEGORY.ALL:
+            if score >= cat.threshold:
+                return cat
+
+        return CATEGORY.UNKNOWN
+
+
+# These would be useful to include in our pages but don't yet see a way to do
```

Comment:
Do you mean just to show what the thresholds are?

Reply:
Mostly no, I meant to be able to define JS variables that contain the threshold for various purposes like coloring or otherwise.